app/controllers/submissions_controller.rb

```diff
@@ -10,6 +10,7 @@ class SubmissionsController < ApplicationController
   before_filter :authorize_only_for_admin,
                 except: [:server_time,
                          :populate_file_manager,
+			 :populate_file_manager_react,
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Tab detected.